os/include/tinyara/clock.h

```diff
@@ -340,6 +340,23 @@ int clock_systimespec(FAR struct timespec *ts);
  * @endcond
  */
 
+/****************************************************************************
+ * Name: initialize_system_time
+ *
+ * Description:
+ *   This function returns initalized system time.
+ *
+ * Parameters:
+ *   tp - Location to return the time
+ *
+ * Return Value:
+ *   OK (0) on success; a negated errno value on failure.
+ *
+ ****************************************************************************/
+#ifdef CONFIG_INIT_SYSTEM_TIME
```

Comment:
This is valid with this? I think it should not.

Reply:
I got it, updated, Could you please reivew again?